app/routes/a.tsx

```diff
@@ -0,0 +1,41 @@
+import { Outlet } from "@remix-run/react";
+import { Main } from "~/components/Main";
+import { db } from "~/db";
+import type { LinksFunction } from "@remix-run/node";
+import { json } from "@remix-run/node";
+import { mostRecentArticles } from "~/modules/articles";
+import styles from "~/styles/front.css";
+import { ArticlesPeek } from ".";
+import { useTranslation } from "react-i18next";
+
+// Set the max number of articles to render
+const MAX_ARTICLES_COUNT = 100;
+
+export const links: LinksFunction = () => {
+  return [{ rel: "stylesheet", href: styles }];
+};
+
+/**
+ * Copied from apps/routes/index.tsx. I can't import this without the functionality breaking
+ */
```

Comment:
Didn't really understand how the loader worked: I thought by importing the loader from index.tsx directly, that the `loader` function would work properly. It did not :( so I had to take copy-paste the function here so that I can use loader in this `a.tsx` file.

Based on your other suggestions, I only have to load the articles & re-defined the loader in this file appropriately